lifetimes/estimation.py

```diff
@@ -278,6 +298,7 @@ def fit(self, frequency, monetary_value, iterative_fitting=4, initial_params=Non
                  This model is not very stable so we suggest >10 for best estimates evaluation.
```

Comment:
The cdnow_customers_transactions data - I wonder if the zeros were the cause... 
(the original docs were included in the initial PR: https://github.com/CamDavidsonPilon/lifetimes/pull/22)